src/components/Header/MobileOptions.tsx

```diff
@@ -79,12 +89,13 @@ export default function MobileOptions() {
         placement="bottom-end"
         content={
           <List>
-            <ListItem>
-              <StyledExternalLink id="stackly-nav-link" href={process.env.REACT_APP_STACKLY_URL as string}>
-                {t('DCA')}
-                <span>↗</span>
-              </StyledExternalLink>
-            </ListItem>
+            {networkWithoutSWPR && (
```

Comment:
Question, doesn't something like this work the same way?
I just quickly tried it and seems to work the same way

```suggestion
            {chainId && !SWPR[chainId] && (
```

Reply:
this seems better but I think It can be done above, having the variable `networkWithoutSWPR` makes it easy to understand.